src/classes/UTIL_FeatureEnablement_TEST.cls

```diff
@@ -56,6 +56,66 @@ private class UTIL_FeatureEnablement_TEST {
         System.assertEquals(true, UTIL_FeatureEnablement.isRecurringDonations2PilotEnabled());
     }
 
+    /**
+     * @description Validates that the isRecurringDonations2PilotEnabled() method can be forced
+     * to return True by using the OverrideFeature_PilotEnabled__c field on the Package_Settings
+     * object.
+     */
+    @isTest
+    private static void shouldReturnTrueWhenPilotIsEnabledWithOverride() {
+        if (!UTIL_Namespace.isScratchOrg()) {
+            return;
+        }
+
+        UTIL_CustomSettingsFacade.getErrorSettingsForTests(
+            new Error_Settings__c(OverrideFeature_PilotEnabled__c = true)
+        );
+
+        System.assertEquals(true,
+            UTIL_FeatureEnablement.isRecurringDonations2PilotEnabled(),
+            'The Pilot should be enabled if this test is running in a scratch org');
+    }
+
+    /**
+     * @description Validates that the isRecurringDonations2PilotEnabled() method can be forced
+     * to return True by using the OverrideFeature_PilotEnabled__c field on the Package_Settings
+     * object.
+     */
+    @isTest
+    private static void shouldReturnFalseWhenPilotIsEnabledWithOverride() {
+        if (!UTIL_Namespace.isScratchOrg()) {
+            return;
+        }
+
+        System.assertEquals(false,
+            UTIL_FeatureEnablement.isRecurringDonations2PilotEnabled(),
+            'The Pilot should be disabled if this test is running in a scratch org');
+    }
+    /**
+     * @description Validates that the isRecurringDonations2PilotEnabled() method can be forced
+     * to return True by using the OverrideFeature_PilotEnabled__c field on the Package_Settings
+     * object.
+     */
+    @isTest
+    private static void shouldRequireValidApiKeyForPilotOverride() {
+        Callable npspApi = (Callable) Type.forName('Callable_API').newInstance();
+        Map<String, Object> params = new Map<String, Object>{
+            'Key' => 'This is an invalid key',
+            'Value' => False
+        };
+
+        try {
+            npspApi.call('Util.ForceOverridePilot', params);
+            System.assert(false, 'A exception should have been thrown for this call');
```

Comment:
Suggestion: We can declare the `Exception actualException;` and then verify exception exists and its message is correct outside of the `try/catch` block.

Reply:
This entire method is being removed.